todo/src/reducers/index.js

```diff
@@ -0,0 +1,53 @@
+import { NEW_TODO, COMPLETER, CLEAR_COMPLETED } from '../actions';
+
+const initialState = localStorage.getItem('todos') === null ? {
+    todos: [
+        {
+            value: 'Functionality',
+            completed: true
+        },
+        {
+            value: 'Style',
+            completed: true
+        }
+    ]
+} : JSON.parse(localStorage.getItem('todos'));
+
+const reducer = (state = initialState, action) => {
+    console.log(action)
+    switch (action.type) {
+        case NEW_TODO:
+            const newTodo = {
+                ...state,
+                todos: [...state.todos, action.payload]
+            }
+            localStorage.setItem('todos', JSON.stringify(newTodo))
+            return newTodo;
+        case COMPLETER:
+            const completer = {
+                ...state,
+                todos: [...state.todos.map(todo => {
+                    if (todo.value === action.payload.value) {
```

Comment:
What's going to happen if there were todos with the same value? Let's say `Mowing Lawn` and `Mowing Lawn`?